types/telegram-web-app/index.d.ts

```diff
@@ -505,6 +531,100 @@ interface HapticFeedback {
     selectionChanged(): void;
 }
 
+interface CloudStorage {
+    /**
+     * A method that stores a value in the cloud storage using the
+     * specified key.
+     *
+     * @param key The key should contain 1-128 characters, only A-Z, a-z, 0-9,
+     * _ and - are allowed.
+     * @param value The value should contain 0-4096 characters. You can store
+     * up to 1024 keys in the cloud storage.
+     * @param callback If an optional callback parameter was passed, the
+     * callback function will be called. In case of an error, the first argument
+     * will contain the error. In case of success, the first argument will be
+     * null and the second argument will be a boolean indicating whether the
+     * value was stored.
+     */
+    setItem(
+        key: string, value: string,
+        callback?: (error: Error | null, success?: boolean) => void,
```

Comment:
`success` contains `true` in case of success and `null` in case of an error
```suggestion
        callback?: (error: Error | null, success: boolean | null) => void,
```

Reply:
Docs don't really specify it this way; I'll verify once more in testing before making the suggested change.

Reply:
Hmm, you're right. If it's not specified in the docs we shouldn't rely on it. How about using `unknown` instead?
```suggestion
        callback?: (error: Error | null, success: boolean | unknown) => void,
```

Reply:
I also just found out that the `error` in case of an error is of type `string` and not `Error`.